diktat-ruleset/src/main/resources/META-INF/version

```diff
@@ -0,0 +1,2 @@
+diktat: ${project.version}
```

Comment:
I believe it'd be better to rename this to smth more unique like `DiktatVersion` or `diktat-version`, because the content of multiple JARs will be "squashed" when creating a far JAR, and file conflicts may occur (i.e. chances are high some dependency JAR will also have `META-INF/version` in its content).

Reply:
Great, thanks!